### ISSUE TITLE
FileNotFoundException (Support sans SSL?)

### ISSUE THREAD

**Issue by marie-bnl** [Litarvan/pronote-api-client-java#1]
Hey ! 

J'aimerais faire un client alternatif pour Pronote sur Android, donc j'ai utilisé cette libraire, mais je n'arrive pas à me connecter à Pronote, à chaque fois ça me donne une erreur `FileNotFoundException`, j'ai essayé les URL suivantes :

- `http://194.167.101.79`
- `http://194.167.101.79/pronote`
- `http://194.167.101.79/pronote/`
- `http://194.167.101.79/pronote/eleve.html`

Précision: voilà l'URL qui est dans le navigateur si je me connecte normalement 
`http://194.167.101.79/pronote/eleve.html?identifiant=XXXXXX`

<img src="https://user-images.githubusercontent.com/39555268/102212113-f0456280-3ed4-11eb-8749-70767d99ee6e.jpg" height="400"/>

Voilà le code : https://hastebin.com/mucosukobi.swift

Les seules parties intéressantes selon moi sont 

                EditText v_pronote_url = getActivity().findViewById(R.id.field_pronote_url);
                String pronote_url = v_pronote_url.getText().toString();

                EditText v_username = getActivity().findViewById(R.id.field_username);
                String username = v_username.getText().toString();

                EditText v_password = getActivity().findViewById(R.id.field_password);
                String password = v_password.getText().toString();

et

                    PronoteAPI api = new PronoteAPI(pronote_url);
                    api.login(new LoginRequest(username, password, pronote_url));

**Comment by marie-bnl** [comment]
Update : 
Si je mets un s donc https://, j'ai plus de FileNotFoundException mais bien une erreur liée à l'absence de certificat SSL, je soupçonne dont un problème dans le support du HTTP non sécurisé

**Comment by foxypiratecove37350** [comment]
C'est vraiment `194.167.101.79` le nom de domaine ? C'est plutôt `[etablissementID].index-education.net` où `[etablissementID]` et ce l'identifiant de l'établissement (ce qu'il y a avent le `.index-education.net`). Fournit nous l'URL dans ce format, car je ne suis pas sûr que tu as Pronote en local et que ce sera plus facile de t'aider.

**Comment by marie-bnl** [comment]
Wow elle date cette issue j’avais oublié ce projet ! Au final je l’ai abandonné malheureusement. Mais non c’était bien `194.167.101.79`, leur instance de Pronote était hébergée sur leurs propres serveurs, pas chez `index-education.net`, donc il y avait pas d’url de la forme `[etablissementID].index-education.net` (bien qu’ils aient fini par se faire héberger là-bas à la fin de l’année).

**Comment by foxypiratecove37350** [comment]
OK, si tu allais sur l'IP tu voyais bien la page depuis l'appareil en question ?

**Comment by marie-bnl** [comment]
Oui

**Comment by foxypiratecove37350** [comment]
Tu as tester il y a peu pour vérifier si ça refonctionne pas ? (avec la nouvelle adresse)

**Comment by marie-bnl** [comment]
Non j’ai abandonné le projet il y a longtemps j’ai plus de quoi tester

**Comment by foxypiratecove37350** [comment]
Bon, je laisse ça comme ça, je pense que l'on a pas besoin de continuer et que c'est du passé maintenant et que tu n'en à plus besoin. Pas besoin de répondre à ce message.

**Comment by marie-bnl** [comment]
Oui, peut-être que quelqu’un aura la même erreur et continuera les tests.
